src/main/java/uk/gov/hmcts/reform/sscs/service/RoboticsService.java

```diff
@@ -69,6 +74,21 @@ public JSONObject createRobotics(RoboticsWrapper appeal) {
         return roboticsAppeal;
     }
 
+    public void attachRoboticsJsonToCaseInCcd(JSONObject roboticsJson, SscsCaseData caseData,
```

Comment:
I would add junit coverage for this method

Reply:
Thanks - that's actually a private method - I think I must have put in an earlier commit when it was still public.